plugin/src/Plugin/GHCSpecter/Task/PrintCore.hs

```diff
@@ -91,21 +105,22 @@ getContent x = (T.pack dtypName, evalue)
       | dtypName == "Unit" =
           let my :: Maybe Unit = cast x
            in Right $ fmap (T.pack . unitString . toUnitId) my
-      | dtypName == "Var" = Right $ getOccNameDynamically (Proxy @Var) x
+      | dtypName == "Var" = Right $ getNameDynamically (Proxy @Var) dflags x
```

Comment:
OccName is not enough for distinguishing Var's. _(uniq symbol) is added with this change.

Reply:
comment added.